3ds2sdk/src/main/kotlin/com/stripe/android/stripe3ds2/service/StripeThreeDs2ServiceImpl.kt

```diff
@@ -198,6 +198,6 @@ class StripeThreeDs2ServiceImpl @VisibleForTesting internal constructor(
     }
 
     private companion object {
-        private const val STRIPE_SDK_REFERENCE_NUMBER = "3DS_LOA_SDK_STIN_020100_00142"
+        private const val STRIPE_SDK_REFERENCE_NUMBER = "3DS_LOA_SDK_STIN_020200_00960"
```

Comment:
This is correct https://drive.google.com/file/d/10iqnXtFIu3tm08e07rf4ygeJPSEzS_Y4/view